plugins/org.python.pydev/tests_analysis/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatementTest.java

```diff
@@ -0,0 +1,235 @@
+package com.python.pydev.analysis.refactoring.quick_fixes;
+
+import org.eclipse.jface.text.Document;
+import org.eclipse.text.edits.ReplaceEdit;
+import org.python.pydev.core.log.Log;
+
+import com.python.pydev.analysis.refactoring.quick_fixes.AddTokenAndImportStatement.ComputedInfo;
+
+import junit.framework.TestCase;
+
+public class AddTokenAndImportStatementTest extends TestCase {
+
+    public String checkImport(String test_document, String test_realImportRep, int test_maxCols) throws Exception {
+
+        Document document = new Document(test_document);
+
+        char trigger = '\n';
+        int offset = 0;
+        boolean addLocalImport = false;
+        boolean addLocalImportsOnTopOfMethod = false;
+        boolean groupImports = true;
+        int maxCols = test_maxCols;
+
+        AddTokenAndImportStatement stmt = new AddTokenAndImportStatement(document, trigger, offset, addLocalImport,
+                addLocalImportsOnTopOfMethod,
+                groupImports, maxCols);
+
+        String realImportRep = test_realImportRep;
+        int fReplacementOffset = document.getLineLength(0);
+        int fLen = 0;
+        String indentString = "";
+        String fReplacementString = "";
+        boolean appliedWithTrigger = false;
+        int importLen = 0;
+
+        ComputedInfo computedInfo = new ComputedInfo(realImportRep, fReplacementOffset, fLen, indentString,
+                fReplacementString, appliedWithTrigger, importLen, document);
+        stmt.createTextEdit(computedInfo);
+
+        for (ReplaceEdit edit : computedInfo.replaceEdit) {
+            try {
+                edit.apply(document);
+            } catch (Exception e) {
+                Log.log(e);
+            }
+        }
+
+        return document.get();
+
+    }
+
+    // normal test
+    public void test1() throws Exception {
+        String test_document = "from math import ceil";
```

Comment:
Naming convention is `camelCase`, so, vars would be `testDocument` and `testRealImportRep`, although in this case I think that the vars should be inlined.